server/game/cards/plots/06/barringthegates.js

```diff
@@ -0,0 +1,14 @@
+const PlotCard = require('../../../plotcard.js');
+
+class BarringTheGates extends PlotCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
```

Comment:
targetController defaults to 'current', while Barring the Gates impacts both players.